### PR TITLE
Adds in Data Validation Utility to rslabel package

### PR DESCRIPTION
**Pull request by ryan-summers**
The data validation utility was previously data_marker.py - However, now it has been added to the rslabel package flow. To validate annotations, run `rslabel mark [annotation_file]`. Help documents are displayed when the script begins.

### CODE REVIEW

File: rslabel_package/rslabel/marker.py [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+            if len(history) > 0:
+                i = history[-1]
+                history = history[:-1]
+            else:
+                print('There is no previous image to display.')
+            continue
+
+        elif status == 's':
+            print('SKIP: {}'.format(annotation['filename']))
+            continue
```
Comment (brennanpeter) [inline rslabel_package/rslabel/marker.py]: Delete continue

File: rslabel_package/rslabel/marker.py [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+    """ Handles annotating and displaying an image.
+
+    Attributes:
+        img: The original OpenCV image. This may be modified by functions.
+        annotations: A list of annotations for the image from Sloth.
+        scale: The scale of the image that is displayed to the user.
+    """
+
+    def __init__(self, img_name, annotations, scale=0.6):
+        """ Initializes the image display.
```
Comment (brennanpeter) [inline rslabel_package/rslabel/marker.py:102]: If Scale could be adjutable that would be sweet. Right now it comes up on my screen a too small

**Review by brennanpeter — CHANGES_REQUESTED** [review]

**Comment by brennanpeter** [comment]
Looks really good except for my 2 pet-peeves :+1: 
I'm not sure how many people we are going to have validating data, but it would be nice to have a system for less experienced validators to be able to say "I'm not sure for this picture, send it to daylan for validation" but that is really just a quality of life thing

**Comment by ryan-summers** [comment]
Inexperienced validators with an image can just choose to skip it :)

**Review by ryan-summers — COMMENTED** [review]
Great points Peter, I'll update it when I wake up tomorrow :)

**Review by brennanpeter — APPROVED** [review]

**Pull request merged**